client/public/index.css

```diff
@@ -0,0 +1,13 @@
+.nav-wrapper {
+    padding: 0 15px;
+}
+.input-field label{
```

Comment:
As I understand you overrides this classes for input labels which affects all forms on the application, but those changes only need for login form on the header. In this case you need restrict this only for class of header.